src/backend/common/models/insight.py

```diff
@@ -62,6 +63,7 @@ class Insight(CachedModel):
         WINNING_MARGIN_DISTRIBUTION: "winning_margin_distribution",
         ELIM_WINNING_MARGIN_DISTRIBUTION: "elim_winning_margin_distribution",
         EINSTEIN_STREAK: "einstein_streak",
+        MATCHES_BY_TEAM: "matches_by_team",
```

Comment:
Might make more sense to name this `matches_played` since it's no longer a dict of `{team: # matches}`